transactions/src/main/java/org/ehcache/transactions/xa/internal/XAStore.java

```diff
@@ -745,7 +745,7 @@ public int rank(final Set<ResourceType> resourceTypes, final Collection<ServiceC
       }
 
       final Store.Provider candidateUnderlyingProvider = selectProvider(resourceTypes, serviceConfigs, xaServiceConfiguration);
-      return 10 + candidateUnderlyingProvider.rank(resourceTypes, serviceConfigs);
+      return 1000 + candidateUnderlyingProvider.rank(resourceTypes, serviceConfigs);
```

Comment:
This was pushed up to make room for `ClusteredStore` on the presumption that an `XAStore` would _wrap_ a `ClusteredStore`.